pkg/operators/v1alpha1/catalogsource_types.go

```diff
@@ -140,6 +141,21 @@ type GrpcPodConfig struct {
 	// +kubebuilder:validation:Enum=legacy;restricted
 	// +kubebuilder:default:=legacy
 	SecurityContextConfig SecurityConfig `json:"securityContextConfig,omitempty"`
+
+	// MemoryTarget configures the $GOMEMLIMIT value for the gRPC catalog Pod. This is a soft memory limit for the server,
+	// which the runtime will attempt to meet but makes no guarantees that it will do so. If this value is set, the Pod
+	// will have the following modifications made to the container running the server:
+	// - the $GOMEMLIMIT environment variable will be set to this value in bytes
+	// - the memory request will be set to this value
+	// - the memory limit will be set to 200% of this value
```

Comment:
Happy to remove this as well, unclear how folks feel about it.

Reply:
I'm happy with that. Also learned some new stuff today: https://weaviate.io/blog/gomemlimit-a-game-changer-for-high-memory-applications